manager/controllers/mockup/policymanager.go

```diff
@@ -40,39 +53,53 @@ func (m *MockPolicyManager) GetPoliciesDecisions(input *policymanager.GetPolicyD
 	case "allow-dataset":
 		// empty result simulates allow
 		// no need to construct any result item
+
 	case "deny-dataset":
 		actionOnDataset := taxonomy.Action{}
-		(&actionOnDataset).Name = "Deny"
+		action := make(map[string]interface{})
+		action["name"] = "Deny"
+		denyAction := map[string]interface{}{}
+		action["Deny"] = denyAction
```

Comment:
Is this required? Isn't the name for Deny action enough?

Reply:
Since we have an explicit action named Deny (like for REdactAction, RemoveAction) as given [here](https://github.com/fybrik/fybrik/blob/67d8886ea0bf58e80c2b1a6999f53c20d6bf2998/samples/taxonomy/example/module/actions.yaml#L25), we need an explicit Deny action related field in the returned action. @roee88